modules/dr-restoring-cluster-state.adoc

```diff
@@ -372,18 +372,22 @@ $ oc get pods -n openshift-ovn-kubernetes -o name | grep ovnkube-node | while re
 $ oc get  pods -n openshift-ovn-kubernetes | grep ovnkube-node
 ----
 
-. Delete and recreate other non-recovery, control plane machines, one by one. After these machines are recreated, a new revision is forced and etcd scales up automatically.
+. Delete and re-create other non-recovery, control plane machines, one by one. After the machines are re-created, a new revision is forced and etcd automatically scales up.
++
+** If you use a bare metal installation, you can re-create a control plane machine by using the same method that you used to originally create it. For more information, see "Installing a user-provisioned cluster on bare metal". 
```

Comment:
I would suggest specifying this as:
`a user-provisioned bare metal installation`
instead of
`a bare metal installation`

Since IPI can be on bare metal and is mentioned in the next bullet.

Reply:
Good suggestion - thanks! I'll update the text.